pkgs/by-name/ri/ripe-atlas-tools/package.nix

```diff
@@ -59,6 +59,8 @@ python3.pkgs.buildPythonApplication rec {
   ];
 
   disabledTests = [
+    # Disable this test because on Python >= 3.12 it fails due to argparse changes https://github.com/python/cpython/pull/124578
```

Comment:
Can you open an issue to upstream to ask them to fix this as well?

Reply:
Documented in https://github.com/RIPE-NCC/ripe-atlas-tools/pull/248.